acceptance/tests/external_facts/non_root_users_default_external_fact_directory.rb

```diff
@@ -0,0 +1,157 @@
+test_name "Non-root default user external facts directory is searched for facts"
+
+confine :except, :platform => 'aix'     # bug FACT-1586
+
+confine :except, :platform => 'windows' # this test is for unix systems
+confine :except, :platform => 'osx'     # does not support managehome
+confine :except, :platform => 'solaris' # does not work with managehome on solaris boxes
+confine :except, :platform => 'eos-'    # does not support user creation ARISTA-37
+
+# TestRail: C64580
+
+require 'facter/acceptance/user_fact_utils'
+extend Facter::Acceptance::UserFactUtils
+
+# Generate an external fact dynamically
+def ext_user_fact(value='BASIC')
+  "test: '#{value}'"
+end
+
+# Retrieve a specific users home directory $HOME/.facter
+#
+def get_user_facter_dir(user_home, platform)
+  if platform =~ /windows/
```

Comment:
At the moment, that is true. I was worried that I might end up moving them to user_fact_util.rb when I need them in a different test and then we'll want  the guard. I guess I want until to add them.